src/LibObjectFile/Elf/ElfObjectFile.cs

```diff
@@ -35,7 +35,7 @@ internal ElfObjectFile(bool addDefaultSections)
         {
             _segments = new List<ElfSegment>();
             _sections = new List<ElfSection>();
-            FileClass = ElfFileClass.Is64;
```

Comment:
Thinking about it, all these fields can be removed from this constructor once it will use `ElfObjectFileExtensions.CopyIndentFrom` 

You can just put a `Encoding = ElfEncoding.Lsb;` in a public constructor (or should it use the encoding returned by `BitConverter.IsLittleEndian`?